README-minishift.md

```diff
@@ -127,49 +123,53 @@ or the command line:
     
     You are logged in as:
         User:     developer
-        Password: <any value>
+        Password: 
```

Comment:
Same comment as above: this removes real command output that I would rather leave there

Reply:
```suggestion
        Password: <any value>
```